rewrite-xml/src/main/java/org/openrewrite/xml/tree/Xml.java

```diff
@@ -669,6 +674,7 @@ public String getPrefix() {
 
         Markers markers;
         Ident name;
+        String documentDeclaration;
```

Comment:
```suggestion
        String documentDeclaration;
        // Override lombok default getter to avoid backwards compatibility problems with old LSTs
        public getDocumentDeclaration() {
            //noinspection ConstantValue
            if ( documentDeclaration == null) {
                return "DOCTYPE"
            }
            return documentDeclaration;
        }
```